introductory/src/listComponent.jsx

```diff
@@ -0,0 +1,8 @@
+export default function List(props){
+    return(
+        <div className="flex gap-3 border-2 w-48 border-slate-950 p-4">
+            <img src={`/${props.name}.jpg`} alt={`${props.name} Image`} className="h-10" />
+            <h1 className='text-2xl'>{props.name}</h1>
```

Comment:
Be **consistency** in choosing which **type of quotation** mark to use between **" "** and **' '**

Reply:
I think there is no need for putting the height of an Image because it will be serving no purpose at all.
But you can feel free to enlighten me if you think otherwise.